app/src/main/kotlin/com/wire/android/ui/common/topappbar/WireCenterAlignedTopAppBar.kt

```diff
@@ -9,13 +9,15 @@ import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.material3.TopAppBarColors
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.unit.Dp
 import com.wire.android.ui.theme.wireDimensions
 import com.wire.android.ui.theme.wireTypography
 
 @Composable
 fun WireCenterAlignedTopAppBar(
     title: String,
+    titleStyle: TextStyle = MaterialTheme.wireTypography.title01,
```

Comment:
Added this title style do we don't need to create another screen/component exactly the same but with a different style.